nominees/opensourceecology.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "OpenSourceEcology",
+  "description": "Creating and sharing open source industrial machine designs",
+  "website": "https://www.opensourceecology.org/",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/OpenSourceEcology/cross-platform-python-gui"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 9,
+      "evidenceText": "It is a project for the development and distribution on web, by Open Source means, of a modular and ecological kit of 50 industrial machine that anybody can download and build on its own",
+      "evidenceURL": "https://www.tedxgeneva.net/talks/jacopo-amistami-open-source-ecology-italy-scrum-methodology-for-a-civilization-starter-kit/#:~:text=Open%20Source%20Ecology%20(OSE)%20is,Village%20Construction%20Set%20(GVCS)."
+    }
+  ],
+  "sectors": [
+    "Engineering",
+    "Infrastructure",
+    "Sustainable Cities"
+  ],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/OpenSourceEcology/cross-platform-python-gui",
+  "organizations": [
+    {
+      "name": "OpenSourceEcology",
```

Comment:
@Msgl could you update the name field to "Open Source Ecology" as represented on the website. Thanks.